src/components/gltf-model-plus.js

```diff
@@ -258,7 +258,7 @@ async function loadGLTF(src, contentType, preferredTechnique, onProgress) {
   const envMap = await CachedEnvMapTexture;
 
   gltf.scene.traverse(object => {
-    object.matrixAutoUpdate = false;
+    object.matrixAutoUpdate = THREE.Object3D.DefaultMatrixAutoUpdate;
```

Comment:
Was confused why we needed this line at all. Suggest adding a comment:
```suggestion
    // GLTFLoader sets matrixAutoUpdate on animated objects, we want to keep the defaults
    object.matrixAutoUpdate = THREE.Object3D.DefaultMatrixAutoUpdate;
```